lib/sub_manager.js

```diff
@@ -39,8 +39,9 @@ SubsManager.prototype.subscribe = function() {
   }
 };
 
-SubsManager.prototype._addSub = function(args) {
+SubsManager.prototype._addSub = function() {
   var self = this;
+  var args = _.toArray(arguments);
```

Comment:
this function is called on line 22 from subscribe(). Now I see my mistake :-). Thanks